src/translations/en.json

```diff
@@ -607,6 +607,37 @@
             }
           }
         },
+        "info": {
```

Comment:
We have a helper in core that we can use to rename keys. https://github.com/home-assistant/core/blob/dev/script/translations/migrate.py#L375-L380

Reply:
We should run it just before merging.